library/aes.c

```diff
@@ -1179,6 +1191,8 @@ int mbedtls_aes_crypt_xts( mbedtls_aes_xts_context *ctx,
     unsigned char tweak[16];
```

Comment:
Again, do we want to update the header documentation to indicate errors are propagated upwards from the ECB operation?

Reply:
same answer